deploy/helm_charts/dc_website/values.yaml

```diff
@@ -55,7 +56,10 @@ mixer:
   githash:
 
   image:
-    repository: gcr.io/datcom-ci/datacommons-mixer
+    registry: "gcr.io"
```

Comment:
registry and repository is not changed, maybe make them constant?

Reply:
done.